### PR TITLE
UICR-163: Provide local translations to ControlledVocab

### PR DESCRIPTION
**Pull request by UladzislauKutarkin**
https://folio-org.atlassian.net/browse/UICR-163
Here we updated props for `ControlledVocab`. Also, added new translations

### CODE REVIEW

File: src/settings/CopyrightStatusSettings.js [hunk 1]
```diff
@@ -29,7 +29,15 @@ class CopyrightStatusSettings extends React.Component {
         baseUrl="coursereserves/copyrightstatuses"
         records="copyrightStatuses"
         label={intl.formatMessage({ id: 'ui-courses.objectName.copyrightstatuses' })}
-        labelSingular={intl.formatMessage({ id: 'ui-courses.objectName.copyrightstatus' })}
+        translations={
+          {
+            cannotDeleteTermHeader: "ui-courses.cv.patronGroup.cannotDeleteTermHeader",
```
Comment (MikeTaylor) [inline src/settings/CopyrightStatusSettings.js:34]: Here and elsewhere, can you please use single quotes (in accordance with standard Stripes style)? `yarn lint` should have told you about this.

File: translations/ui-courses/en.json [hunk 2]
```diff
@@ -192,5 +192,10 @@
     "addFastAddItem.title": "Add Fast Add item",
     "addFastAddItem.tooltip": "Create a new Inventory record using 'Fast Add' and add it to this course as a reserve",
     "errors.mustBeInteger": "This value must be an integer",
-    "SENTINEL": "SENTINEL"
+    "SENTINEL": "SENTINEL",
```
Comment (MikeTaylor) [inline translations/ui-courses/en.json:195]: Please make sure SENTINEL remains the last entry. (That's what it's for.)

File: translations/ui-courses/en.json [hunk 3]
```diff
@@ -192,5 +192,10 @@
...
+    "cv.patronGroup.cannotDeleteTermHeader": "Cannot delete the patron group",
```
Comment (MikeTaylor) [inline translations/ui-courses/en.json:196]: The use of `patronGroup` in the names of these translations cannot possible be right.

**Review by MikeTaylor — CHANGES_REQUESTED** [review]
Thanks for seeing to this. Aside from the lint issues (single vs. double quote), the names of the translation tags, and the position of the `SENTINEL` translation, it looks good.

**Review by vashjs — APPROVED** [review]
Please fix eslint problems, other then that looks good to me.

File: src/settings/CopyrightStatusSettings.js (hunk 1 above)
Comment (UladzislauKutarkin) [reply src/settings/CopyrightStatusSettings.js]: fixed! Thanks

File: translations/ui-courses/en.json (hunk 2 above)
Comment (UladzislauKutarkin) [reply translations/ui-courses/en.json]: Done!

File: translations/ui-courses/en.json (hunk 3 above)
Comment (UladzislauKutarkin) [reply translations/ui-courses/en.json]: Removed  `patronGroup` from key.

**Review by MikeTaylor — APPROVED** [review]
Perfect, thank you!

**Pull request merged**